tests/cypress/support/commands.js

```diff
@@ -275,8 +275,9 @@ Cypress.Commands.add('goToPolicyDetailsPage', (policyName, namespace='default',
 // must be run on policy details page
 Cypress.Commands.add('goToPolicyClusterPage', (policyName, policyConfig, clusterName) => {
   var namespace = policyConfig['namespace']
-  cy.waitUntil(() => cy.get('.one-cluster-status').children('a').should('exist'), {'interval': 1000, 'timeout':60000})
-  cy.get('.one-cluster-status').children('a').contains(clusterName).click()
+  cy.get('.one-cluster-status').children('a').contains(clusterName)
```

Comment:
I kept getting this failure
<img width="451" alt="image" src="https://user-images.githubusercontent.com/16092291/113422175-41049c80-939a-11eb-85ea-3c24ff56faa2.png">

Reply:
I had to change to use {force: true} in https://github.com/open-cluster-management/grc-ui/pull/541/commits/bded1ea4356cc932108fc3e675b8d905ab662109#diff-42424b3ade03fe1a7a13e099f2adc6ee7aefac801afdad7fd5eaf4416fcda75aR280

Reply:
It looks like a second `should` was added here. By using `have.attr`, the second `should` is returning the URL string of the attribute instead of the DOM element since it considers it a context change (See https://docs.cypress.io/api/commands/should#Yields).